BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeDeepLinkActivity.java

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+/**
+ * This Activity is used by the SDK Activity to enable default deep link handling and should not
+ * be used directly.
+ */
+public class BraintreeDeepLinkActivity extends AppCompatActivity {
```

Comment:
Is the reason this is public so merchants can perform any clean-up on this activity if need be?

Reply:
This is a good callout we did a quick test. It does need to be public, I think because it's a deep link target the Android system needs access to the class definition in order to instantiate it.